common/determined_common/experimental/experiment.py

```diff
@@ -99,3 +99,6 @@ def top_n_checkpoints(
                 t_ids.add(ckpt["trialId"])
 
         return checkpoint_refs[:limit]
+
+    def __repr__(self) -> str:
+        return "Experiment(id={})".format(self.id)
```

Comment:
I think this will be more useful if we print the experiment fields.  is there a to_json method that we can call then print?

Reply:
The python SDK is really limited right now. The experiment object can only query checkpoints. It doesn't have any information about the experiment itself. We, in the near term, want to autogenerate these from protobuf so I'm hesitant to add more functionality right now. Can we move forward with this method for now?

Reply:
that makes sense